list-singleton.cabal

```
@@ -7,10 +7,10 @@ description:
   The @list-singleton@ package allows you to easily and clearly create lists
   with only one element in them, which are typically called "singleton lists"
   or simply "singletons". (Not to be confused with
-  the [@singletons@](https://hackage.haskell.org/package/singletons) package!)
+  the [singletons](https://hackage.haskell.org/package/singletons) package!)
```

The fix for this particularly inconvenient case I discovered recently is to move `@` outside the `[]`. I hope it works here as well...
```suggestion
  the @[singletons](https://hackage.haskell.org/package/singletons)@ package!)
```

---

Oh, neat! So it's the exact opposite of Markdown 😆

---

Yes thank you, that works! Updated the PR.